pkg/test/istio.io/tasks/traffic-management/ingress/secure-ingress-sds/cleanup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [[ -d "mtls-go-example" ]]; then
```

Comment:
Remove the `if`s. `rm -rf` will work regardless.

Reply:
done. thanks.